build/teamcity/internal/release/process/make-and-publish-build-artifacts.sh

```diff
@@ -52,14 +52,14 @@ git tag "${build_name}"
 tc_end_block "Tag the release"
 
 tc_start_block "Compile and publish artifacts"
-BAZEL_SUPPORT_EXTRA_DOCKER_ARGS="-e TC_BUILDTYPE_ID -e TC_BUILD_BRANCH=$build_name -e gcs_credentials -e gcs_bucket=$gcs_bucket" run_bazel << 'EOF'
+BAZEL_SUPPORT_EXTRA_DOCKER_ARGS="-e TC_BUILDTYPE_ID -e TC_BUILD_BRANCH=$build_name -e build_name=$build_name -e gcs_credentials -e gcs_bucket=$gcs_bucket" run_bazel << 'EOF'
```

Comment:
I'm curious. Why make `build_name` lowercase here? 

Also, why not just use `TC_BUILD_BRANCH` given they both have the same value?

Reply:
> I'm curious. Why make build_name lowercase here?

I didn't "make it lowercase" -- the variable already exists and has a lowercase name. 🤷 Just keeping it the same way it is.

> Also, why not just use TC_BUILD_BRANCH given they both have the same value?

Out of an abundance of caution I am not doing this. I am not sure if the value of `TC_BUILD_BRANCH` from the host or the container would be used because bash scripts are confusing. (The values differ between the host and container for some reason.)